core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.spec.ts

```diff
@@ -634,6 +644,25 @@ describe('EditEmaEditorComponent', () => {
                     expect(spectator.query(byTestId(testId))).not.toBeNull();
                 });
             });
+
+            it('should open seo results when clicking on a social media tile', () => {
+                // We only support VTL
+
+                store.load({
+                    url: 'index',
+                    language_id: '3',
+                    'com.dotmarketing.persona.id': DEFAULT_PERSONA.identifier
+                });
+                spectator.detectChanges();
+
+                const deviceSelector = spectator.debugElement.query(
+                    By.css('[data-testId="dot-device-selector"]')
```

Comment:
`byTestId`?

Reply:
I needed the `debugElement` the query method for debug elements doesn't support  `byTestId` spectator method